src/main/java/com/enterprise4045/groceryplanner/GroceryPlannerController.java

```diff
@@ -145,4 +148,24 @@ public String searchItemsForm(@RequestParam(value="searchTerm", required = false
     public String superCoolPage() {
         return "superCoolPage";
     }
+
+    @GetMapping("/itemNameAutoComplete")
+    @ResponseBody
+    public List<LabelValue> itemNameAutoComplete(@RequestParam(value = "term",
+            required = false, defaultValue = "") String term){
+        List<LabelValue> allItemNames = new ArrayList<LabelValue>();
+        try {
+            List<Item> items = loggedItemService.fetchItems(); //Should have term in the brackets
+            for (Item item: items) {
+                LabelValue labelValue = new LabelValue();
+                labelValue.setLabel(item.getDescription());
```

Comment:
I just changed the item.toString() to item.getDescription() so that it looked more user friendly in the drop down